jkq-ddsim/README.md

```diff
@@ -0,0 +1,26 @@
+# JKQ-DDSIM Benchmark
+
+This benchmark uses Google Benchmark to measure the runtime of the jkq-ddsim simulator.
+
+For more information visit [the university site](http://iic.jku.at/eda/research/quantum_simulation) or the [GitHub repo](https://github.com/iic-jku/ddsim).
+
+## Short Explanation
```

Comment:
I'm wondering if this is not very fair to include it in this benchmark (maybe we should have another benchmark for other type of simulators rather than full amplitude simulation), since all the framework is only benchmarked in full amplitude simulation.

Reply:
I thought about this too. Having both approaches in the same benchmark has the advantage that one can easily compare different approaches and see their strenghts/weaknesses. For example, while our DD approach does really well for the single gate benchmarks, it absolutely tanks for the quasi-random QCBM. (In fact, I reduced the range to 4..21 qubits as the 22 qubit instance is running for more than 48h on my server...)
Maybe we can have an explaination in the readme/the results to emphasize that there are (as of now) two different types of simulation?

Reply:
I agree plotting all in one image would be easier to compare, but I think if anyone wants to compare their work with others they could always draw a relative plot (like ours) and highlight their own framework, but here we would like to keep things as neutral as possible, a better idea might be using a dynamic widget to only show selected benchmarks.  But that could be an enhancement to add later.

Reply:
Interactive plots are a great idea! I don't think this is possible with markdown, but in a long shot this could be done via github pages.

To make this more obvious on the results page, I added a column to the package info table with the "type of simulator". I think this is a good solution for now.